### ISSUE TITLE
Messages in user's language

### ISSUE THREAD

**Issue by ghost** [jtauber-archive/django-notification#28]
Users expect to get e-mails in their native languages. 
So the notification should somehow know and use user's language. 
Does django-notification have this feature?

**Comment by jezdez** [comment]
Yes, there is a function called get_notification_language() (http://github.com/jtauber/django-notification/blob/master/notification/models.py#L199). It will use a setting `NOTIFICATION_LANGUAGE_MODULE` to load the user language from an arbitrary model and should be formatted like `<app>.<model>`. The model should have two fields: a foreignkey to the User class (`user`) and a language field (`language`) which stores the language code (e.g. `en`).

In Pinax for example we use this model:
http://github.com/pinax/pinax/blob/7118dc9367c75de47be80a6ced2b4afc0dceb573/pinax/apps/account/models.py#L16

**Comment by ghost** [comment]
Thank you! Now it's clear. Probably you will want to add this comment to the documentation.

**Comment by bjunix** [comment]
I think there should also be a fallback to the default language code:
LANGUAGE_CODE defined in settings.py

What do you think? I just put up a quick fix at my fork: http://github.com/bjunix/django-notification/commit/da1a81fda93e7c1cd743d0b5f2eac217c05c8668
